payment/build.gradle

```diff
@@ -26,6 +26,7 @@ android {
     release {
       minifyEnabled true
       proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-payment.pro'
+      consumerProguardFiles 'consumer-proguard-payment.pro'
```

Comment:
I highly recommend not to do this

Reply:
Let me know the reason or reference why not to do this? I could not use the lib as it is documented in README. What would be the recommended steps to include the SDK in my app?

Reply:
You can just add @Keep annotation for constants

Reply:
Yes, I added @keep annotations for all the public and their protected classes as a general rule. Can you review the consumer-proguard-payment.pro from my PR? I found a typical library's proguard rules as mentioned in here:
https://stuff.mit.edu/afs/sipb/project/android/sdk/android-sdk-linux/tools/proguard/docs/index.html#manual/examples.html
The thing is I can simply add @keep for all the classes and constants, but the lib doesn't stop here, maybe, other things will be available for users in future. So a general proguard rule for a typical library from MIT should not hurt anyone.

Reply:
https://youtu.be/aBm5iYg7uJU?t=1485

Reply:
I got your point, what about the latest commit? The docs on[ README.md](https://github.com/PaycomUZ/AndroidSDK/blob/master/README.md) says one can use the following classes: 
[PaymentActivity.java](https://github.com/PaycomUZ/AndroidSDK/blob/master/payment/src/main/java/uz/paycom/payment/PaymentActivity.java) constants, [PaycomSandBox.java](https://github.com/PaycomUZ/AndroidSDK/blob/master/payment/src/main/java/uz/paycom/payment/utils/PaycomSandBox.java) and [Result.java](https://github.com/PaycomUZ/AndroidSDK/blob/master/payment/src/main/java/uz/paycom/payment/model/Result.java).
I tried to make it as defensive as possible:

```
-keep class uz.paycom.payment.PaymentActivity {
    public static <fields>;
}

-keep class uz.paycom.payment.utils.PaycomSandBox { *; }
-keep class uz.paycom.payment.model.Result { *; }
```